app/scripts/controllers/courtSearchInfo.ctrl.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
This file really should have a .spec.js file to test the functionality in this file. Again, it's fine for now